pkg/controller/constraint/constraint_controller.go

```diff
@@ -161,7 +161,7 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 				return reconcile.Result{Requeue: true}, nil
 			}
 		}
-		log.Info("instance will be added", "instance", instance)
+		log.Info("constraint updated", "instance", instance)
```

Comment:
This should say `handling constraint update` for clarity

Reply:
done!